modules/bio_files_processor_module.py

```diff
@@ -0,0 +1,111 @@
+import os
+
+
+def path_output(input_fasta: str, output_fasta: str):
+    path = '/'.join(input_fasta.split("/")[:-1])
+    if output_fasta == '':
+        return os.path.join(path, 'output_' + input_fasta.split("/")[-1])
+    else:
+        return os.path.join(path, output_fasta)   
+
+
+def write_fasta_file(lines_fasta: list, output_file: str):
+    os.makedirs(os.path.dirname(output_file), exist_ok=True)
+    with open(output_file, mode='w') as output_fasta:
+        for line_number in range(len(lines_fasta)-1):
+            if lines_fasta[line_number].startswith('>'):
+                output_fasta.write(lines_fasta[line_number])
+            elif lines_fasta[line_number+1].startswith('>'):
+                output_fasta.write(lines_fasta[line_number])
+            else:
+                output_fasta.write(lines_fasta[line_number].strip('\n'))
+        output_fasta.write(lines_fasta[len(lines_fasta)-1].strip('\n'))
+
+
+def find_protein(lines_file: list):
+    list_protein = []
+    for line_number in range(len(lines_file)):
+        if lines_file[line_number].startswith('Query #'):
+            position = lines_file[line_number+3].find('Scientific')
+            list_protein.append(lines_file[line_number+5][:position])
+    return list_protein
+
+def write_output_protein_file(list_protein: list, output_protein_file: str):
+    os.makedirs(os.path.dirname(output_protein_file), exist_ok=True)
+    list_protein = sorted(list_protein, key=lambda s: s.casefold())
+    with open(output_protein_file, mode='w') as output_file:
+        for protein in list_protein:
+            output_file.write(protein + '\n')
+
+
+def find_genes_proteins(input_gbk: str):
+    with open(input_gbk) as input_file:
+        lines_file = input_file.readlines()
+    proteins_list = []
+    genes_list = []
+    for line_number in range(len(lines_file)):
+        if lines_file[line_number].find('/gene=') != -1:
+            gene_name = lines_file[line_number].split('"')[1]
+            seq = ''
+            line_number_protein_start = line_number + 1
+            while lines_file[line_number_protein_start].find('/translation') == -1:
+                line_number_protein_start += 1
+            line_number_protein_end = line_number_protein_start + 1
+            while lines_file[line_number_protein_end].find('"') == -1:
+                line_number_protein_end += 1
+            seq += lines_file[line_number_protein_start].split('"')[1].strip()
+            for number in range(line_number_protein_start + 1, line_number_protein_end):
+                seq += lines_file[number].strip()
```

Comment:
```suggestion
            for line in lines[line_number_protein_start + 1:line_number_protein_end]:
                seq += line
```

возможно ещё подсократить имена переменных (типа `ln_` вместо `line_number_`), но это уже на любителя)